lib/msal-common/src/request/ScopeSet.ts

```diff
@@ -199,4 +247,15 @@ export class ScopeSet {
         }
         return "";
     }
+
+    /**
+     * Prints scopes into a space-delimited string
+     * @param scopes
+     */
+    static finalScopes(scopes: Array<string>): string {
```

Comment:
this is a duplicate of printScopes

Reply:
Not needed if you are using ScopeSet correctly.

Reply:
Yep, moved over to using printScopes